### DISCUSSION TITLE
New Here & Some Questions!

### DISCUSSION THREAD

**Discussion by chicknnsoup** [nzp-team/nzportable#768] [General]
Hey all, total newbie here! First off, I wanna say how insanely impressive it is that NZP exists and is so accessible on so many platforms! I found out about this project a few weeks back when I found an old build for the PSP and it's insane how dedicated the community is.

Anyways, I've seen the threads about mapping tutorials and might start looking into it soon. I have a few questions though about mapping, and I'm so sorry if they've been answered a million times hahaha...

1. Is it currently possible to retexture assets such as player/ai/weapon models?
2. Is it possible to import player/ai/weapon models I make myself?
3. How's the future of NZP looking? Will there be more official updates & new content?
4. Not a question but I love how easy it is to swap sounds out :P 

Sorry again if these questions have been answered a bunch, I'm kinda new to this and Github as a whole... and thanks for your hard work, NZP team!

**Comment by bigjackson** [comment]
1 and 2 are both possible, although i feel like the game should allow for a custom folder for models for maps to utilize.

**and i can't answer question 3 because i'm not on the team lol**
